varnish-sys/src/validate.rs

```diff
@@ -1,10 +1,15 @@
 use crate::ffi;
 use crate::ffi::{
-    director, req, sess, vcldir, vfp_ctx, vfp_entry, vrt_ctx, ws, DIRECTOR_MAGIC, REQ_MAGIC,
-    SESS_MAGIC, VCLDIR_MAGIC, VCL_BACKEND, VFP_CTX_MAGIC, VFP_ENTRY_MAGIC, VRT_CTX_MAGIC, WS_MAGIC,
+    director, req, sess, vrt_ctx, ws, DIRECTOR_MAGIC, REQ_MAGIC,
+    SESS_MAGIC, VCL_BACKEND, VRT_CTX_MAGIC, WS_MAGIC,
 };
+#[cfg(not(feature = "_lts_60"))]
+use crate::ffi::{vcldir, vfp_ctx, vfp_entry, VCLDIR_MAGIC, VFP_CTX_MAGIC, VFP_ENTRY_MAGIC};
+
+#[cfg(not(feature = "_lts_60"))]
 use crate::vcl::{DeliveryFilters, FetchFilters};
 
+#[cfg(not(feature = "_lts_60"))]
```

Comment:
i guess we should move a lot of these to a new file to simplify gating

Reply:
I haven't look at that yet, but yes, that would be nice